corehq/util/doc_processor/couch.py

```diff
@@ -82,6 +85,7 @@ class CouchProcessorProgressLogger(ProcessorProgressLogger):
     :param doc_types: List of doc_types that are being processed
     """
     def __init__(self, doc_types):
+        super(CouchProcessorProgressLogger, self).__init__()
```

Comment:
Not going to hold the PR on this, but I was wondering about something (maybe you can teach me something): my mental model of `super()` vs `super(ExplicitClassName, self)` was that basically in our code base since upgrading to python 3 `super()` is just always preferred. Is there some situation in which they're meaningfully different and `super(ExplicitClassName, self)` is preferred?

Reply:
`super()` is always preferred. The only place where `super(ExplicitClassName, self)` should be used (except in old code) is outside of `ExplicitClassName` class definition, in which case it is not only preferred but semantically required. Thankfully that's very rare.

Reply:
old habbits: https://github.com/dimagi/commcare-hq/pull/27091